blockcode/README.md

```diff
@@ -2,58 +2,65 @@
+To do this, we encapsulate everything that is specific to the turtle language into one file (`turtle.js`) that we can easily swap with another file. Nothing else should be specific to the turtle language; the rest should just be about handling the blocks (`block.js` and `menu.js`) or be generally useful web utilities (`util.js`, `drag.js`, `file.js`). That is the goal, although to maintain the small size of the project, some of those utilities are less general-purpose and more specific to their use with the blocks.
 
-One thing that struck me when writing a block language is that the language is its own IDE. You can't just code up blocks in your favourite text editor, the IDE has to be designed and developed in parallel with the block language. This has some pros and cons. On the plus side, everyone will use a consistent environment, there is no room for religious wars around what editor to use. On the downside, it can be a huge distraction from building the block language itself.
+One thing that struck me when writing a block language was that the language is its own IDE. You can't just code up blocks in your favourite text editor; the IDE has to be designed and developed in parallel with the block language. This has some pros and cons. On the plus side, everyone will use a consistent environment and there is no room for religious wars about what editor to use. On the downside, it can be a huge distraction from building the block language itself.
 
 ### The Nature of Scripts
 
-A Blockcode script, like a script in any language (whether block or text-based) is a sequence of operations that will be followed and which make up the program. In the case of Blockcode the script consists of HTML elements which are iterated over and which are each associated with a particular JavaScript function to run when that block's turn comes. Some blocks can contain (and are responsible for running) other blocks, and some blocks can contain numeric values which are passed to the JavaScript functions as arguments.
+A Blockcode script, like a script in any language (whether block- or text-based), is a sequence of operations to be followed. In the case of Blockcode the script consists of HTML elements which are iterated over, and which are each associated with a particular JavaScript function which will be run when that block's turn comes. Some blocks can contain (and are responsible for running) other blocks, and some blocks can contain numeric values which are passed to the JavaScript functions as arguments.
```

Comment:
I can see it going either way. There is the program as run and the program as written, it has a verb/noun duality. So I was saying that the blocks were both, but I agree it didn't read well and is probably not necessary. Which is a long way of saying I like your version better.